internal/provisioner/plain/controllers/bundle_controller.go

```diff
@@ -392,3 +393,67 @@ func (r *BundleReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&corev1.ConfigMap{}).
 		Complete(r)
 }
+
+func (r *BundleReconciler) bundleImagePod(pod *corev1.Pod, source rukpakv1alpha1.ImageSource) *corev1.Pod {
+	if len(pod.Spec.InitContainers) != 1 {
+		pod.Spec.InitContainers = make([]corev1.Container, 1)
+	}
+
+	pod.Spec.InitContainers[0].Name = "install-unpacker"
```

Comment:
nit: Should we make a function that is specifically for setting up the `UnpackImage`? We call this logic twice.

Reply:
Sure, good catch. No need to duplicate code.